core/Pos/Binary/Core/Coin.hs

```diff
@@ -73,6 +73,20 @@ encode (unsafeGetCoin -> w) = encodeVarint mega ++ encodeVarint (reversedBase10
   where
     (mega, micros) = w `divMod` 1000000
 
+size :: Coin -> Int
```

Comment:
And we'd need to rename `encode` and `decode` too.

~to be honest I kinda don't want to change this code because it's Vincent code and touching it feels wrong~